### PR TITLE
Fixing code style for pinot-minion/pinot-perf/pinot-tools/pinot-integration-tests

### PR DESCRIPTION
**Pull request by xiangfu0**
## Description
- Fixing pinot-minion
- fixing pinot-perf
- fixing pinot-tools
- fixing pinot-integration-tests

## Upgrade Notes
Does this PR prevent a zero down-time upgrade? (Assume upgrade order: Controller, Broker, Server, Minion)
* [ ] Yes (Please label as **<code>backward-incompat</code>**, and complete the section below on Release Notes)

Does this PR fix a zero-downtime upgrade introduced earlier?
* [ ] Yes (Please label this as **<code>backward-incompat</code>**, and complete the section below on Release Notes)

Does this PR otherwise need attention when creating release notes? Things to consider:
- New configuration options
- Deprecation of configurations
- Signature changes to public methods/interfaces
- New plugins added or old plugins removed
* [ ] Yes (Please label this PR as **<code>release-notes</code>** and complete the section on Release Notes)

### CODE REVIEW

File: pinot-tools/src/main/java/org/apache/pinot/tools/anonymizer/ArrayBasedGlobalDictionaries.java [hunk 1]
```diff
@@ -187,6 +187,7 @@ public int compare(Object o1, Object o2) {
               return ((Double) o1).compareTo((Double) o2);
             }
           };
+          break;
```
Comment (xiangfu0) [inline pinot-tools/src/main/java/org/apache/pinot/tools/anonymizer/ArrayBasedGlobalDictionaries.java:190]: bug found

File: pinot-tools/src/main/java/org/apache/pinot/tools/SpeedTest.java [hunk 2]
```diff
@@ -32,40 +32,43 @@
...
  */
 public class SpeedTest {
-  static int SIZE = 10000000;
-  static String FILE_NAME = "/tmp/big_data.dat";
-  static double readPercentage = 0.9;
-  static int[] heapStorage;
-  static ByteBuffer directMemory;
-  static MappedByteBuffer mmappedByteBuffer;
-  static int[] readIndices;
+  private SpeedTest() {
```
Comment (Jackie-Jiang) [inline pinot-tools/src/main/java/org/apache/pinot/tools/SpeedTest.java:35]: This doesn't seem a util class. Change the static methods to non-static?

File: pinot-tools/src/main/java/org/apache/pinot/tools/SpeedTest.java [hunk 3]
```diff
@@ -32,40 +32,43 @@
...
+  }
+
+  static final int SIZE = 10000000;
```
Comment (Jackie-Jiang) [inline pinot-tools/src/main/java/org/apache/pinot/tools/SpeedTest.java:38]: Add `private`?

File: pinot-tools/src/main/resources/conf/sample_perf_benchmark.yaml [hunk 4]
```diff
@@ -17,7 +17,7 @@
 # under the License.
 #
 
-!!org.apache.pinot.tools.perf.PerfBenchmarkDriverConf
+  !!org.apache.pinot.tools.perf.PerfBenchmarkDriverConf
```
Comment (Jackie-Jiang) [inline pinot-tools/src/main/resources/conf/sample_perf_benchmark.yaml:20]: Is this correct?

**Review by Jackie-Jiang — APPROVED** [review]

**Pull request merged**